packages/lab/src/radio-button/RadioButtonIcon.tsx

```diff
@@ -29,10 +29,10 @@ export const RadioButtonIcon = ({
       viewBox="0 0 14 14"
       width="14"
     >
-      <circle className={withBaseName("border")} cx="7" cy="7" r="6.5" />
+      <circle className={withBaseName("circle")} cx="7" cy="7" r="6.5" />
```

Comment:
@libertymayc this is the 6.5 value referenced in css

Reply:
can this radius be documented in the size specs @navkaur76 if it makes sense to? if it relates more strongly to the svg though maybe its unnecessary